easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb

```
@@ -1549,7 +1549,7 @@ exts_list = [
         'checksums': ['0bd2dcd4ee1915448d325fb5e66bee68e0497cbd91ef67a11b400b2fbe52ff59'],
     }),
     ('lhs', '1.1.1', {
-        'checksums': ['903e9f2adde87f6f9ad41dd52ff83d28a645dba69934c7535142cb48f10090dc'],
+        'checksums': ['00278a65b18c86799301d27c55e3d32c6c307aed745ea69c9b96bbb1d0a352c2'],
```

@guimou Please (consistently) change to accept *both* checksums as valid (like we're already doing

```suggestion
        'checksums': [('903e9f2adde87f6f9ad41dd52ff83d28a645dba69934c7535142cb48f10090dc',
                       '00278a65b18c86799301d27c55e3d32c6c307aed745ea69c9b96bbb1d0a352c2')],
```

---

I've applied this fix myself in 64677cab42, so we can get this PR merged quickly...

---

Lol, thanks a lot!